app/assets/javascripts/discourse/app/initializers/ember-events.js

```diff
@@ -1,36 +1,14 @@
 let initializedOnce = false;
 
 /**
- * By default Ember listens to too many events. This tells it the only events
- * we're interested in. (it removes mousemove, touchstart and touchmove)
+ * EventDispatcher events have been trimmed to the minimal list required.
+ *
+ * TODO: Investigate why remaining events are required.
  *
  * @type {Record<string, string>}
  */
-export const EVENTS = {
-  touchend: "touchEnd",
-  touchcancel: "touchCancel",
-  keydown: "keyDown",
-  keyup: "keyUp",
-  keypress: "keyPress",
-  mousedown: "mouseDown",
-  mouseup: "mouseUp",
-  contextmenu: "contextMenu",
+const EVENTS = {
   click: "click",
```

Comment:
> Need to investigate why / how the `click` event still needs to be handled by the EventDispatcher.

Any luck working out why this is still required? Would be nice to know, even if we end up keeping it 🤔